browser/brave_content_browser_client.cc

```diff
@@ -352,9 +354,7 @@ void MaybeBindSkusSdkImpl(
 
 }  // namespace
 
-BraveContentBrowserClient::BraveContentBrowserClient()
-    : session_token_(base::RandUint64()),
```

Comment:
any headers we can remove here for rand int?

Reply:
removed in https://github.com/brave/brave-core/pull/12234/commits/752b22430a0b52d30ca0769ec43dfe6ee41f4adf